### ISSUE TITLE
Application of agent policies fail in Terraform with WSL

### ISSUE THREAD

**Issue by philippjh** [terraform-google-modules/terraform-google-cloud-operations#40]
### TL;DR

In order to install Google's [Ops Agent](https://cloud.google.com/blog/products/operations/ops-agent-now-ga-and-it-includes-opentelemetry) service to monitor my VMs, I was trying to define the required agent policy within my Terraform configuration (see below). However, `terraform apply` did not finish successfully on my Windows 21H2 machine running WSL with Ubuntu 20.04.4 LTS indicating a dubious "file or directory not found" error.

### Expected behavior

Terraform applys the given agent policy to my GCP project.

### Observed behavior

```
╷
│ Error: External Program Execution Failed
│
│   with module.agent_policy.module.gcloud-destroy.data.external.env_override[0],
│   on .terraform/modules/agent_policy.gcloud-destroy/main.tf line 73, in data "external" "env_override":
│   73:   program = ["${path.module}/scripts/check_env.sh"]
│
│ The data source received an unexpected error while attempting to execute the program.
│
│ Program: .terraform/modules/agent_policy.gcloud-destroy/scripts/check_env.sh
│ Error: fork/exec .terraform/modules/agent_policy.gcloud-destroy/scripts/check_env.sh: no such file or directory
╵
╷
│ Error: External Program Execution Failed
│
│   with module.agent_policy.module.gcloud-upsert.data.external.env_override[0],
│   on .terraform/modules/agent_policy.gcloud-upsert/main.tf line 73, in data "external" "env_override":
│   73:   program = ["${path.module}/scripts/check_env.sh"]
│
│ The data source received an unexpected error while attempting to execute the program.
│
│ Program: .terraform/modules/agent_policy.gcloud-upsert/scripts/check_env.sh
│ Error: fork/exec .terraform/modules/agent_policy.gcloud-upsert/scripts/check_env.sh: no such file or directory
╵
```

While the output suggests, that the scripts are not available, they actually are. Running one of the "missing" scripts, leads to the following error:

```
.terraform/modules/agent_policy.gcloud-destroy/scripts/check_env.sh: bad interpreter: /bin/sh^M: no such file or directory
``` 

This error indicates an issue with erroneous control sequences / line terminators in the Shell scripts. And indeed, I was able to fix this issue by running `dos2unix` on both `agent_policy.gcloud-upsert/scripts/check_env.sh` and `agent_policy.gcloud-destroy/scripts/check_env.sh`, for example:

```
philipp@machine:tmp/infrastructure ‹main*›$ file .terraform/modules/agent_policy.gcloud-upsert/scripts/check_env.sh
.terraform/modules/agent_policy.gcloud-upsert/scripts/check_env.sh: POSIX shell script, ASCII text executable, with CRLF line terminators
philipp@machine:tmp/infrastructure ‹main*›$ dos2unix .terraform/modules/agent_policy.gcloud-upsert/scripts/check_env.sh
dos2unix: converting file .terraform/modules/agent_policy.gcloud-upsert/scripts/check_env.sh to Unix format...
philipp@machine:tmp/infrastructure ‹main*›$ file .terraform/modules/agent_policy.gcloud-upsert/scripts/check_env.sh
.terraform/modules/agent_policy.gcloud-upsert/scripts/check_env.sh: POSIX shell script, ASCII text executable
```

I suggest, that these files are created with Unix line endings by default. However, I have no idea, where these files are actually created, since they are (apparently?) not part of the repository. 

### Terraform Configuration

```hcl
module "agent_policy" {
  source     = "terraform-google-modules/cloud-operations/google//modules/agent-policy"
  version    = "0.2.3"

  project_id = "my-project-id"
  policy_id  = "ops-agents-example-policy"
  agent_rules = [
    {
      type               = "ops-agent"
      version            = "current-major"
      package_state      = "installed"
      enable_autoupgrade = true
    },
  ]

  os_types = [
    {
      short_name = "ubuntu"
      version    = "20.04"
    },
  ]
}
```

### Terraform Version

```sh
Terraform v1.2.3
on linux_amd64
+ provider registry.terraform.io/hashicorp/external v2.2.2
+ provider registry.terraform.io/hashicorp/google v4.10.0
+ provider registry.terraform.io/hashicorp/google-beta v4.17.0
+ provider registry.terraform.io/hashicorp/null v3.1.1
+ provider registry.terraform.io/hashicorp/random v3.3.1
```

**Comment by philippjh** [comment]
Actually, there were more files than what I've mentioned with CRLF line endings. I resorted to converting them all with the following command from the root of my Terraform directory:

```
find .terraform/modules/agent_policy* -name "*.sh" -exec dos2unix {} \;
```
